scripts/ruff.toml

```diff
@@ -0,0 +1,4 @@
+extend = "../pyproject.toml"
+ignore = [
+    "D", # Only limited docstrings are required.
```

Comment:
In this project we will have to include more docstrings to keep information about what tests do there.